app/scripts/background.js

```diff
@@ -394,13 +396,18 @@ function overrideContentSecurityPolicyHeader() {
 let connectRemote;
 let connectExternalExtension;
 let connectExternalCaip;
+let connectRemoteCaip;
 
 browser.runtime.onConnect.addListener(async (...args) => {
   // Queue up connection attempts here, waiting until after initialization
   await isInitialized;
 
   // This is set in `setupController`, which is called as part of initialization
   connectRemote(...args);
+
+  if (process.env.MULTICHAIN_API && !isManifestV3) {
```

Comment:
i think `! isManifestV3` is a reasonable proxy for non-chromium build. Otherwise we have to check user agents or something?